src/internal/base-component/component-metrics.ts

```diff
@@ -19,8 +18,8 @@ export function useComponentMetrics(componentName: string, { packageSource, pack
       metrics.sendMetricOnce('awsui-viewport-width', window.innerWidth || 0);
       metrics.sendMetricOnce('awsui-viewport-height', window.innerHeight || 0);
     }
-    metrics.logComponentLoaded();
-    metrics.logComponentUsed(componentName.toLowerCase());
+    metrics.logComponentsLoaded();
```

Comment:
Renamed the function, because it has nothing to do with a particular component, it does not even know its name. This call reports the entire bundle being loaded.